gdx/src/com/badlogic/gdx/graphics/Pixmap.java

```diff
@@ -112,6 +112,7 @@ public static Pixmap createFromFrameBuffer (int x, int y, int w, int h) {
 	final Gdx2DPixmap pixmap;
 	int color = 0;
 
+	public boolean ready = false;
```

Comment:
If this is public and does something (anything), it should be documented, because I have no idea what this is supposed to do.

Reply:
I changed to private and created a method `isReady()`. It is only useful when Pixmap is loaded async (currently only GWT).